packages/ace-cli/package.json

```diff
@@ -18,8 +18,10 @@
   "main": "lib/index.js",
   "bin": "bin/ace.js",
   "dependencies": {
+    "@daisy/ace-config": "^1.0.0",
```

Comment:
Do we need an explicit dependency to `ace-config`, since we're transitively depending on it via `ace-core`?

Reply:
Yes, a package should explicitly declare all its direct dependencies.